src/main/plugin/iso19139.ca.HNAP/duplicate-metadata.xsl

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (C) 2001-2016 Food and Agriculture Organization of the
+  ~ United Nations (FAO-UN), United Nations World Food Programme (WFP)
+  ~ and United Nations Environment Programme (UNEP)
+  ~
+  ~ This program is free software; you can redistribute it and/or modify
+  ~ it under the terms of the GNU General Public License as published by
+  ~ the Free Software Foundation; either version 2 of the License, or (at
+  ~ your option) any later version.
+  ~
+  ~ This program is distributed in the hope that it will be useful, but
+  ~ WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU General Public License
+  ~ along with this program; if not, write to the Free Software
+  ~ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
+  ~
+  ~ Contact: Jeroen Ticheler - FAO - Viale delle Terme di Caracalla 2,
+  ~ Rome - Italy. email: geonetwork@osgeo.org
+  -->
+
+<xsl:stylesheet   xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="2.0"
+                  xmlns:gco="http://www.isotc211.org/2005/gco"
+                  xmlns:gmd="http://www.isotc211.org/2005/gmd"
+                  xmlns:xlink="http://www.w3.org/1999/xlink"
+                  exclude-result-prefixes="#all">
+
+  <xsl:output method="xml" indent="yes"/>
+
+  <xsl:variable name="doiName"
+                select="'Digital Object Identifier (DOI)'"/>
+
+  <xsl:template match="/root">
+    <xsl:apply-templates select="*[1]"/>
+  </xsl:template>
+
+
+  <!-- Remove DOI identifiers -->
+  <xsl:template match="gmd:identifier[
+                                contains(*/gmd:code/*/text(), 'datacite.org/doi/')
+                                or contains(*/gmd:code/*/text(), 'doi.org')
+                                or contains(*/gmd:code/*/@xlink:href, 'doi.org')]" />
+
+  <!-- Remove DOI links -->
+  <xsl:template match="gmd:onLine[*/gmd:name/gco:CharacterString = $doiName]" />
```

Comment:
Could you add a comment indicating why this was not simply an import like the following and why we are not matching based on cit:protocol like in iso19139.
 
```
<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="2.0">
  <xsl:import href="../iso19139/duplicate-metadata.xsl"/>
</xsl:stylesheet>
```

Reply:
@ianwallen added a comment to explain the custom HNAP change.